public/app/plugins/panel/geomap/editor/layerEditor.tsx

```diff
@@ -93,12 +93,14 @@ export function getLayerEditor(opts: LayerEditorOptions): NestedPanelOptions<Map
         // TODO -- add opacity check
       }
 
-      builder.addBooleanSwitch({
-        path: 'tooltip',
-        name: 'Display tooltip',
-        description: 'Show the tooltip for layer',
-        defaultValue: true,
-      });
+      if (opts.category === ['Data layer']) {
+        builder.addBooleanSwitch({
+          path: 'tooltip',
+          name: 'Display tooltip',
+          description: 'Show the tooltip for layer',
+          defaultValue: true,
+        });
+      }
```

Comment:
I'm not seeing this option appearing for inside settings of individual data layers.

I'm not sure having a toggle on a layer by layer basis for tooltips makes a lot of sense vs. a single global enable / disable tooltip setting (maybe as part of `Panel options` or potentially even `Map controls`?) @ryantxu thoughts on this?
![Screen Shot 2022-03-29 at 12 50 03 PM](https://user-images.githubusercontent.com/22381771/160695338-296a8870-5e06-4296-ab7e-666519f7ccf4.png)

Reply:
Yes, that's not right, the tooltip should've been there for data layer. Looking into it.

Reply:
Your latest change works great! On re-review, having tooltip disable / enable on a layer by layer basis makes sense to me now, though I still think it might make sense to have a global option as well. We can revisit this another time though